bee-common/bee-packable-derive/src/lib.rs

```diff
@@ -23,7 +30,7 @@ use syn::parse_macro_input;
 #[proc_macro_error]
 #[proc_macro_derive(Packable, attributes(packable))]
 pub fn packable(input: TokenStream) -> TokenStream {
-    let input = parse_macro_input!(input);
+    let input: DeriveInput = parse_macro_input!(input);
```

Comment:
Just curious, why is this needed ?

Reply:
I think it is not, is just debris from my refactor.